### PR TITLE
Account linking policy

### PR DESCRIPTION
**Pull request by dinkom**
Pull request for: https://github.com/stormpath/stormpath-sdk-python/issues/292

### CODE REVIEW

File: tests/mocks/test_account_linking_policy.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+try:
+    from mock import MagicMock
+except ImportError:
+    from unittest.mock import MagicMock
+
+
+class TestAccountLinkingPolicy(TestCase):
+
+    def test_modifying_account_linking_policy(self):
```
Comment (dinkom) [inline tests/mocks/test_account_linking_policy.py]: @rdegges Should I change this? I've modelled it after an existing test which is very similar, but maybe that one was written before there were codacy checks so it remained intact.

Comment (dinkom) [inline tests/mocks/test_account_linking_policy.py]: @rdegges Changed it

**Comment by rdegges** [comment]
Can you update this PR to fix the merge conflicts? It looks like the resources/__init__.py needs to be corrected to support both the manual account linking resources and these ones =)

**Comment by dinkom** [comment]
@rdegges Conflicts resolved :)

**Pull request merged**